src/lang/pt-br.json

```diff
@@ -9,103 +9,104 @@
 	"ACTION.SHOVE": "Derrubar",
 	"ACTION.UNARMED_STRIKE": "Ataque desarmado",
 
-	"ACTOR.TypeCharacter": "Personagem",
-	"ACTOR.TypeMonster": "Monstro",
-	"ACTOR.TypeStronghold": "Fortaleza",
-
-	"ARMOR.BODY": "Corpo",
-	"ARMOR.HELMET": "Elmo",
-	"ARMOR.PART": "Parte do Corpo",
-	"ARMOR.RATING": "Valor de Armadura",
-	"ARMOR.SHIELD": "Escudo",
-	"ARMOR.TOTAL": "Total",
-
-	"ARTIFACT.APPEARANCE": "Aparência",
-	"ARTIFACT.ARTIFACT_BONUS": "Bônus de Artefato",
-	"ARTIFACT.DESCRIPTION": "Descrição",
-	"ARTIFACT.DRAWBACK": "Inconveniente",
-	"ARTIFACT.EPIC": "Épico",
-	"ARTIFACT.LEGENDARY": "Lendário",
-	"ARTIFACT.MIGHTY": "Poderoso",
-	"ARTIFACT.REGULAR": "Item normal",
-
-	"ATTACK.BLUNT": "Esmagamento",
-	"ATTACK.FEAR": "Medo",
-	"ATTACK.OTHER": "Outros",
-	"ATTACK.SLASH": "Cortante",
-	"ATTACK.STAB": "Golpear",
-
-	"ATTRIBUTE.AGILITY": "Agilidade",
-	"ATTRIBUTE.EMPATHY": "Empatia",
-	"ATTRIBUTE.STRENGTH": "Força",
-	"ATTRIBUTE.WITS": "Astúcia",
```

Comment:
The attributes should not be all caps. If they are the main tab on the character sheet will overflow and scroll as a result.